csrc/ir/utils.cpp

```diff
@@ -797,7 +797,8 @@ std::vector<TensorView*> getTVsWithDynamicTransform(Fusion* fusion) {
 CompareDomainResult compareDomains(
     std::vector<IterDomain*> dom0,
     const std::vector<IterDomain*>& dom1,
-    const std::vector<IterDomain*>& additional_ids) {
+    const std::vector<IterDomain*>& additional_ids,
+    bool ignore_broadcast) {
```

Comment:
Broadcast IDs also need to be considered to detect extra IDs added by `setLoopDomain`.